posts/2022-09-01-22.0.0.10-beta.adoc

```diff
@@ -0,0 +1,581 @@
+// change the SUB_TAG_1/2/3 to the heading tags
+//
+// However if there's only 1 new feature, delete the previous section and change it to the following sentence:
+// "The link:{url-about}[Open Liberty] 22.0.0.10-beta includes SUB_FEATURE_TITLE"
+// // // // // // // //
+
+// https://github.com/OpenLiberty/open-liberty/issues/22166
+[#java]
+== Java 19 Support
+Java 19 is coming soon, and with it the following features and changes:
+
+* 405: Record Patterns (Preview)
+* 422: Linux/RISC-V Port
+* 424: Foreign Functions & Memory API (Preview)
+* 425: Virtual Threads (Preview)
+* 426: Vector API (Fourth Incubator)
+* 427: Pattern Matching for switch (Third Preview)
+* 428: Structured Concurrency (Incubator)
+
+By taking advantage of trying out the new changes in Java 19 now, you will get more time to review your applications, microservices and runtime environments which means you will be a step ahead when it becomes generally available.
+
+To give this a try, link:https://jdk.java.net/19/[download the early access release of Java 19], download and install the link:https://openliberty.io/downloads/#runtime_betas[22.0.0.10-beta] version of Open Liberty, edit your Liberty server's link:https://openliberty.io/docs/22.0.0.10/reference/config/server-configuration-overview.html#server-env[server.env] file to point JAVA_HOME to your Java 19 installation and start testing!
+
+For more information on Java 19, visit the Java 19 link:https://jdk.java.net/19/release-notes[release notes page], link:https://download.java.net/java/early_access/jdk19/docs/api/[API Javadoc page] or link:https://openliberty.io/docs[download page].
+For more information on Open Liberty, please visit our documentation page.
+
+As work is made towards full Java 19 support, there may functionality that might not be 100% ready yet but is in progress to be complete soon!
+
+// https://github.com/OpenLiberty/open-liberty/issues/22092
+[#xml]
+== XML Binding 4.0
+Jakarta XML Binding 4.0 is an API for automating the mapping between XML documents and Java objects. Support provides the updated XML Binding 4.0 API, it's Reference Implementation, and introduces the xmlBinding-4.0 feature to Open Liberty.
+
+The xmlBinding-4.0 feature provides the Jakarata XML Binding 4.0 tools for marshalling and unmarshing XML to Java: xjc and schemagen. Both command line tools preform identically to the Jakarata EE 9 tooling.
+
+To enable the new Jakarta XML Binding 4.0 feature, add the xmlBinding-4.0 feature to your `server.xml`. Here's an example of what the feature configuration looks like:
+
+[source,xml]
+----
+    <featureManager>
+          <feature>xmlBinding-4.0</feature>
+    </featureManager>  
+----
+
+For more information see the link:https://jakarta.ee/specifications/xml-binding/[Jakarta XML Binding Specification]
+
+// https://github.com/OpenLiberty/open-liberty/issues/21974
+[#bell]
+== BELL SPI Visibility and BELL Properties

```

Comment:
This section seems like a whole blog post embedded into this release blog post. Should it be broken out into a separate post and then just have a summary in this beta release blog post that links to it instead?

Reply:
It definitely could be shortened by removing the repeated statements.